src/App.tsx

```diff
@@ -1,61 +1,108 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/todo';
+import { User } from './types/user';
+
+function getUser(userId: number): User | null {
+  const foundedUser = usersFromServer.find(user => user.id === userId);
+
+  return foundedUser || null;
+}
+
+const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+const users: User[] = usersFromServer.map(user => ({
+  ...user,
+}));
 
 export const App = () => {
+  const [userSelect, setUserSelect] = useState('');
+  const [titleInput, setTitleInput] = useState('');
+  const [isSubmit, setIsSubmit] = useState(false);
+
   return (
     <div className="App">
       <h1>Add todo form</h1>
 
-      <form action="/api/users" method="POST">
+      <form
+        action="/api/users"
+        method="POST"
+        onSubmit={(event) => {
+          event.preventDefault();
+          setIsSubmit(true);
+          if (userSelect && titleInput) {
+            const selectedUser = users.find(user => user.name === userSelect);
```

Comment:
you have getUser function and can use it here
also use state for store your data